### ISSUE TITLE
macOS ARM64 / Apple Silicon

### ISSUE THREAD

**Issue by EduardoRFS** [revery-ui/revery#998]
- Operating System: macOS
- Revery Version: trunk
- OCaml version: 4.10.1
- Native, Bytecode, or JS build: Native

This is mostly to track the requirements from the Revery side to the new macOS using ARM64

## Known Issues

- [ ] esy-sdl2 needs to be updated

**Comment by EduardoRFS** [comment]
Now that we're on 4.12, the compiler is already there. Will be checking which deps are needed

**Comment by EduardoRFS** [comment]
seems like esy-sdl2 is still the missing piece

**Comment by zbaylin** [comment]
Yeah @EduardoRFS I made a PR in esy-sdl2 a while back to upgrade us to 2.0.14 (which worked for me on my M1 Mac Mini) -- apparently there are some issues with how I rebased with Wayland which I haven't gotten around to fixing. It seems pretty simple if you want to look at it! revery-ui/esy-sdl2#19

**Comment by zbaylin** [comment]
I also am worried that we need to rebase esy-skia as well, which might be a nightmare 😨

**Comment by snadon** [comment]
As a M1 owner and proud user of Onivim2, I was just wondering what is the current status for this issue? 🙏

**Comment by syaiful6** [comment]
yes, I tried to build revery in Apple M1, it failed because SKIA compiled with x86, I made a PR in esy-skia to fix that https://github.com/revery-ui/esy-skia/pull/38 with that PR I am able to build Revery natively in M1.
